expandFns.php

```diff
@@ -221,6 +221,17 @@ function title_capitalization($in, $caps_after_punctuation) {
     create_function('$matches', 'return mb_strtolower($matches[0]);'),
     trim($new_case)
   );
+  
+  $new_case = preg_replace_callback("/ (L')([a-zÀ-ÿ]+)/", function($matches) { /** French l'Words  **/
```

Comment:
Do we also need to match unicode alternatives to the apostrophe, i.e. &rsquo; and kin?